Plugin.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
ehm, random bom?

Reply:
most probably, not sure what's up wit that.